src/pages/createOrderNewFlow.js

```diff
@@ -127,6 +127,7 @@ class CreateOrderPage extends React.Component {
       stepPayment: null,
       error: null,
       stripe: null,
+      email: '',
```

Comment:
This isn't necessary anymore ;)

Reply:
Oops! forgot to remove that :slightly_smiling_face: